static/js/components/PersonalTab.js

```diff
@@ -30,7 +35,7 @@ class PersonalTab extends React.Component {
       <Grid className="profile-tab-grid">
         <Cell col={1} />
         <Cell col={10}>
-          <PersonalForm {...this.props} />
+          <PersonalForm {...this.props} validator={personalValidation} />
```

Comment:
Yeah exactly, when we're on the profile we want to validate the UI stuff (the switches and all that) but we don't use the switches on the users page.